### PR TITLE
remove WSS connection, replace w/ a hearbat app…

### PR DESCRIPTION
**Pull request by matzew**
changes as discussed !

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM :taco:

**Review by sjwoodman — APPROVED** [review]

File: knative-kubecon/README.md [hunk 1]
```diff
@@ -328,16 +329,9 @@ spec:
 
 This is starting to get a little bit more interesting. This EventSource is a so called `ContainerSource`. As such, it runs a container
 based off the image given and instructs it to send its event to the sink described in the YAML. In this case, this container happens
-to be a Websocket connector and we want all events coming from the specified Websocket server to be forwarded to the given sink. That
+to be a _hearbeat_ application, written in Golang, which generates events at a configurable rate, to be forwarded to the given sink. That
```
Comment (jcrossley3) [inline knative-kubecon/README.md]: can we link to the source repo here?

**Comment by matzew** [comment]
@jcrossley3  I've added the link to the `main.go` file - good point!

Also, I added a step required by @markito to show how to clone _this_ repo

**Pull request merged**